src/application/controllers/load-guardian-name.ts

```diff
@@ -0,0 +1,26 @@
+import { type GetGuardianName } from '@/domain/use-cases'
+import { success, type HttpRequest, type HttpResponse, serverError } from '../helpers'
+import { type Controller } from '../protocols'
+
+export class LoadGuardianNameController implements Controller {
```

Comment:
Get or Load? These words have similar meaning. Words like `load` and `save` immediatly remind me database transactions, `get` is more general to things like this. Similarly we used add to both context, database transactions and use cases, what you think?

Reply:
When I wrote this controller I left "get" and "load" because there was a conflict with the same name for a reason I don't remember now. But I wrote this code late at night and maybe my reasoning was a bit off.

I can change the dependency to "LoadGuardianName" to get the sense closer to the controller. What do you think?

Reply:
Make sense. I like it.